simtools/data_model/metadata_collector.py

```diff
@@ -462,7 +467,7 @@ def _process_metadata_from_file(self, meta_dict):
             meta_dict["cta"]["product"]["description"] = self._remove_line_feed(
                 meta_dict["cta"]["product"]["description"]
             )
-        except KeyError:
+        except (KeyError, AttributeError):
```

Comment:
None of the errors are in the docstring yet. Despite being an internal function, I prefer the docstring to be correct.

Reply:
I also don't see when an attribute error could be raised by this function. Can you please clarify?

Reply:
These exceptions are passed. Maybe I am wrong, and but don't we document only Exceptions which a module raises?

On AttributeError: self._remove_line_feed(None) is giving this Exception. This is also tested in https://github.com/gammasim/simtools/blob/d9920d2536bf3371a8052a7d96c82e8b9ecd41b8/tests/unit_tests/data_model/test_metadata_collector.py#L258

Reply:
Yes, you are right. I oversaw this. If we `pass` we don't need the statements.